### PR TITLE
cloud: update roadmap description 

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-roadmap.md [hunk 1]
```diff
@@ -158,21 +228,26 @@ The TiDB Cloud roadmap brings you what's coming in the near future, so you can s
...
+    <td>Support key rotation on TiDB clusters for AWS.</td>
+    <td>Support key rotation on TiDB clusters to improve the security of encrypted data.</td>
   </tr>
   <tr>
-    <td>Encrypted backup and restore</td>
-    <td>Support encrypted backup and restore.</td>
-    <td>Provide a method to securely back up and restore a database.</td>
+    <td>Key management</td>
+    <td>Support making your own key manageable (BYOK from AWS).</td>
+    <td>Allow customers to use their own data encryption keys on AWS.</td>
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-roadmap.md:245]:
```suggestion
    <td>Allow you to use your own data encryption keys on AWS.</td>
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: tidb-cloud/tidb-cloud-roadmap.md [hunk 2]
```diff
@@ -158,21 +228,26 @@ The TiDB Cloud roadmap brings you what's coming in the near future, so you can s
...
+  </tr>
+   <tr>
+    <td>Audit logging</td>
+    <td>Enhance the database audit logging feature.</td>
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-roadmap.md:249]:
```suggestion
    <td>Enhance the database audit logging.</td>
```

**Review by easonn7 — APPROVED** [review]

**Pull request merged**